docs/source/getting_started/install.rst

```diff
@@ -1,14 +1,14 @@
 .. _install:
 
 =====================
-5. Install the plugin
+2. Install the plugin
 =====================
 
 Here we will simply install the plugin before continuing.
 
 .. include:: ../../../README.rst
    :start-line: 54
-   :end-line: 74
+   :end-line: 71
```

Comment:
```suggestion
   :end-line: 81
```

I think a few lines are missing. Maybe worth repeating ourselves here, as now adding more content to the README.rst can easily break this page.